packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -145,6 +173,20 @@ function insistMeterID(m) {
   Nat(BigInt(m.slice(1)));
 }
 
+export const getAllStaticVats = kvStore => {
+  const result = [];
+  for (const k of enumeratePrefixedKeys(kvStore, 'vat.name.')) {
+    const name = k.slice(9);
+    const vatID = kvStore.get(k) || Fail`getNextKey ensures get`;
+    result.push([name, vatID]);
+  }
```

Comment:
Eliminating a magic number:
```suggestion
  const prefix = 'vat.name.';
  for (const k of enumeratePrefixedKeys(kvStore, prefix)) {
    const name = k.slice(prefix.length);
    const vatID = kvStore.get(k) || Fail`getNextKey ensures get`;
    result.push([name, vatID]);
  }
```